03 Tests samples/src/components/MessageList/onContextMenu.test.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
Please try to do this generic to only acces as config file and not need to call and configure this in every test file